build/release.sh

```diff
@@ -3,6 +3,9 @@ set -e # Any subsequent(*) commands which fail will cause the shell script to ex
 
 SBT_CMD="sbt"
 
+SBT_2_12="sbt -Dspark.include=true ++$(sed -n '5p' < .travis.yml | cut -c5-12)"
+SBT_2_11="sbt ++$(sed -n '4p' < .travis.yml | cut -c5-12)"
```

Comment:
Could you use system properties here? Reading the travis config is too fragile and easy to break

Reply:
The idea was to keep scala versions at the single place, however it is not possible to retrieve both of them with travis. e.g., release job will have only one $TRAVIS_SCALA_VERSION